client/src/components/mainBoxComponent.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default function Box({ children, ...props}) {
```

Comment:
This component seems like it does nothing. You don't need it as it's just a div.